dsp-tools/dsp-atlantis/values.yaml

```diff
@@ -6,6 +10,8 @@ azure:
 vault:
   azure:
     pathPrefix: secret/devops/terraform/azure
+  github:
+    pathPrefix: /secret/suitable/github/dsp-atlantis
```

Comment:
I am not sure if the leading `/` will work with secrets-manager... At least we tend to omit it in most of our configs

Reply:
derp, typo, fix'd